src/app/core/services/transaction.service.ts

```
@@ -292,9 +294,9 @@ export class TransactionService {
     } else if (fyleCategory === 'per diem') {
       vendorDisplayName = expense.tx_num_days.toString();
       if (expense.tx_num_days > 1) {
-        vendorDisplayName += ' Days';
+        vendorDisplayName += ' ' + this.translocoService.translate('services.transaction.days');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)